src/components/common/YourCompany/YourCompany.vue

```diff
@@ -271,17 +256,21 @@
     </template>
 
     <!-- Recognition Date and Time (alterations and BEN corrections only) -->
-    <div class="section-container" v-if="isAlterationFiling || isBenIaCorrectionFiling">
+    <template v-if="isAlterationFiling || isBenIaCorrectionFiling">
+
+    <v-divider class="mx-4 my-1" />
```

Comment:
This code block should now be indented.

Reply:
Done

Reply:
The v-divider should be indented too.

Reply:
And you don't need the blank line before the divider (line 260) but it would be nice to have a blank line after it (for readability).

Reply:
Done

Reply:
Done